src/xdocs/writingchecks.xml

```diff
@@ -20,9 +20,9 @@
 
       <p>
         OK, so you have finally decided to write your own Check.  Welcome
-        aboard, this is really a easy thing to do. Very basic Java knowledge is required
-        to write a Check, it is good practice for even for student.
-        There are actually two
+        aboard: this is a really easy thing to do. Very basic Java knowledge is required
+        to write a Check. It is good practice even for students.
+        Actually, there are two
         kinds of Checks, so before you can start, you have to find out
         which kind of Check you want to implement.
```

Comment:
I am thinking this sentence need to be redone.

`There are actually two different types of checks that can be made, each with their own requirements. Before you start coding, you will have to find out which kind of Check you want to implement.`

Reply:
Rereading this again: we never actually mention what these two types of checks are. I assumed initially that this was FileSetCheck and AbstractCheck, but that seems a bit off still. Could you clarify?

Reply:
In proposed by rnveach there is no reference to exact type of Checks, so I agree with rnveach